packages/core/contracts/polygon/OracleChildTunnel.sol

```diff
@@ -103,6 +103,16 @@ contract OracleChildTunnel is OracleBaseTunnel, OracleAncillaryInterface, FxBase
         return lookup.price;
     }
 
+    /**
+     * @notice Generates stamped ancillary data in the format that it would be used in the case of a price request.
+     * @param ancillaryData ancillary data of the price being requested.
+     * @param requester sender of the initial price request.
+     * @return the stamped ancillary bytes.
+     */
+    function stampAncillaryData(bytes memory ancillaryData, address requester) public view returns (bytes memory) {
```

Comment:
I don't think we need to audit this method since its view-only

Reply:
I agree, but if another contract were to depend on the validity of the output, it might be audit-worthy. Probably won't be too hard to audit, though since they'll already be reviewing _stampAncillaryData. In my opinion, we should just toss it over to them just to be safe.